subcommands/config/rotate_certs.go

```diff
@@ -0,0 +1,75 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/foundriesio/fioctl/subcommands"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	rotateCmd := &cobra.Command{
+		Use:   "rotate-certs",
+		Short: "Rotate device x509 keypairs in this group used to connect to the device gateway",
+		Run:   doCertRotate,
+		Long: `This command will send a fioconfig change to a device to instruct it to perform
+a certificate rotation using the EST server configured with "fioctl keys est".
+
+This command will only work for devices running LmP version 90 and later.
+
+It's highly recommend to read the product documentation for this feature before
+running it against production devices.`,
+	}
+	cmd.AddCommand(rotateCmd)
```

Comment:
It literally means that we encourage the admin to rotate certificates across all factory devices at once.

Is it an acceptable behavior for us? Maybe, restrict it to only group and device levels?

Reply:
sorry, I missed that ist is required.